web/packages/teleport/src/Console/index.tsx

```diff
@@ -27,6 +29,15 @@ export default function Index() {
     return new ConsoleContext();
   });
 
+  React.useEffect(() => {
+    async function initUserContext() {
+      const user = await userService.fetchUserContext();
+      ctx.setStoreUser(user);
+    }
+
+    initUserContext();
+  }, []);
```

Comment:
what if this failed to fetch? i think it's better to move this in the console to handle the error since we need to know the username for a feature

Reply:
probably sim to this if you trace the `init` call: https://github.com/gravitational/teleport/blob/851cb6598ee9da31adf735204d1ead64082b3e1e/web/packages/teleport/src/teleportContext.tsx#L68

Reply:
Sure. I moved it into console, as well as wrapped in added a `useAttempt`.